README.md

```diff
@@ -4,13 +4,18 @@ MIR (acronym for MyCoRe/MODS Institutional Repository) is an open source reposit
 
 ## Installation instructions
 Detailed instructions you can find on [MIR Documentation site](http://mycore.de/generated/mir)
- - starting up a servlet container in development environment is quite simple: `mvn install -am -pl mir-webapp && mvn -Pdev -Djetty org.codehaus.cargo:cargo-maven2-plugin:run -pl mir-webapp` If you want to test the application with Tomcat instead replace `-Djetty` by `-Dtomcat=9`
  - initialize solr configuration using `git submodule update --init --recursive`
+ - to start solr, go to mir-webapp
+  - install solr with the command: `mvn solr-runner:copyHome solr-runner:installSolrPlugins`
+  - run solr with the command: `mvn solr-runner:start`
+  - stop solr with the command: `mvn solr-runner:stop`
+  - update solr with the command: `mvn solr-runner:stop solr-runner:copyHome solr-runner:start`
+ - starting up a servlet container in development environment is quite simple: `mvn install -am -pl mir-webapp && mvn -Pdev -Djetty org.codehaus.cargo:cargo-maven2-plugin:run -pl mir-webapp` If you want to test the application with Tomcat instead replace `-Djetty` by `-Dtomcat=9`
```

Comment:
Missing info: you need to run this command from mir folder(instead of mir-webapp)